deps-build/.gitignore

```diff
@@ -0,0 +1,2 @@
+win32/
+win64/
```

Comment:
Deleted the binaries for Windows

It takes about 20 min to build from scratch. Not too bad.